frontend/src/test/navMenu.spec.js

```diff
@@ -0,0 +1,27 @@
+import {mount} from '@vue/test-utils'
+import NavMenu from '@/components/navMenu.vue'
+
+// tests by Kailey Bales
+
+// test the navigation menu component
+describe('NavMenu.vue', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(NavMenu);
+  });
+  test('renders a navigation menu', () => {
+    //check if the navigation menu is rendered
+    expect(wrapper.find('v-list').exists()).toBe(true)
```

Comment:
I would actually check to see if the list item is rendered. I believe you can check for the title of the item.